crates/ide-completion/src/completions/flyimport.rs

```diff
@@ -315,8 +317,10 @@ fn import_on_the_fly_pat_(
             .into_iter()
             .filter(ns_filter)
             .filter(|import| {
-                !ctx.is_item_hidden(&import.item_to_import)
-                    && !ctx.is_item_hidden(&import.original_item)
+                let item = &import.item_to_import;
+                !ctx.is_item_hidden(item)
```

Comment:
Did you want to do `!ctx.is_item_hidden(&import.original_item)` here?

Reply:
Oops, you're right! Thanks for catching this. Will file another PR shortly!